src/Client/Notification.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
+class Notification extends AbstractClient
+{
+    public function getNotifications(
+        ?string $seen = null,
+        ?int $skip = null,
+        ?int $take = null
+    ): \BTCPayServer\Result\NotificationList {
+        $url = $this->getApiUrl() . 'users/me/notifications/?';
+
+        //add a seen value if set
+        if (isset($seen)) {
```

Comment:
here is probably a good case to use http_build_query() like I mentioned in the other PR

Reply:
`$queryParameters = [
            'seen' => $seen,
            'skip' => $skip,
            'take' => $take
        ];

        $url .= http_build_query($queryParameters);`

Reply:
$queryParameters = [
            'seen' => $seen,
            'skip' => $skip,
            'take' => $take
        ];

        $url .= http_build_query($queryParameters);

Reply:
Fixed it.